packages/url/src/test/index.test.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
There's no place for random comments on unrelated code in a pull request, they should not never be added.

Comments such as this original comment should be added as a comment on the PR or a new issue created but never random comments on random lines of code of a pull request, its beyond confusing IMHO.